modules/nf-core/samtools/calmd/main.nf

```diff
@@ -0,0 +1,33 @@
+process SAMTOOLS_CALMD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::samtools=1.15.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.15.1--h6899075_1':
+        'quay.io/biocontainers/samtools:1.15.1--h6899075_1' }"
+
+    input:
+    tuple val(meta), path(bam)
+    path fasta
```

Comment:
Is FASTA necessary? I don't see it in the clamd docs?

http://www.htslib.org/doc/samtools-calmd.html

Reply:
Also you've not listed in the meta.yml

Reply:
I haven't been able to get the example in the samtools docs to work. In the latest versions the usage looks like this:

`
Usage: samtools calmd [-eubrAESQ] <aln.bam> <ref.fasta>
`